src/index.html

```diff
@@ -5,8 +5,397 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class=" nav__item">
+            <a class="is-active nav__link" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a data-qa="nav-hover" class="nav__link" >Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <article class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        >
+        <div class="card__info">
+          <h2 class="card__title">
+            <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+          </h2>
+          <p class="card__product-code">
```

Comment:
```suggestion
          </h2>
          
          <p class="card__product-code">
```
Add one empty line between sibling blocks inside the same parent block for both of them.

Reply:
Same about similar cases below